libc-test/build.rs

```diff
@@ -341,7 +341,9 @@ fn main() {
 
             "getrlimit" | "getrlimit64" |    // non-int in 1st arg
             "setrlimit" | "setrlimit64" |    // non-int in 1st arg
-            "strerror_r" if linux => true,   // actually xpg-something-or-other
+            "strerror_r" |                   // actually xpg-something-or-other
+            "sendmmsg" | "recvmmsg"          // glibc header has wrong declarations
```

Comment:
It's `unsigned int flags` vs `int flags`.  I filed a bug at https://sourceware.org/bugzilla/show_bug.cgi?id=19936